sdkit/__init__.py

```diff
@@ -78,3 +79,12 @@ def vram_usage_level(self, level):
             self.vram_optimizations = {"KEEP_FS_AND_CS_IN_CPU", "SET_ATTENTION_STEP_TO_16"}
         elif level == "high":
             self.vram_optimizations = {"SET_ATTENTION_STEP_TO_2"}
+
+    @property
```

Comment:
How about keeping this as a regular variable, and do the bool cast when creating the compel object?